packages/next/server-actions/index.ts

```diff
@@ -0,0 +1,80 @@
+'use server';
+
+import { type GetContextParameters, type InteractionMode } from '@logto/node';
+import NodeClient from '@logto/node/edge';
+
+import BaseClient from '../src/client';
+import { createSession } from '../src/session';
+import type { LogtoNextConfig } from '../src/types.js';
+
+export type { LogtoContext, InteractionMode } from '@logto/node';
+
+export default class LogtoClient extends BaseClient {
+  constructor(config: LogtoNextConfig) {
+    super(config, {
+      NodeClient,
+    });
+  }
+
+  handleSignIn = async (
```

Comment:
better to use the traditional definition since we cannot bind `this` for arrow functions? this may limit some use cases